examples/material-ui/README.md

```diff
@@ -0,0 +1,53 @@
+# Material-UI Example
```

Comment:
This README doesn’t follow the other examples style, for example it doesn’t have a button to try it in CodeSandbox